src/bundle/Resources/public/js/scripts/helpers/system.helper.js

```diff
@@ -0,0 +1,78 @@
+(function (global, doc, ibexa) {
+    const { userAgent } = window.navigator;
+    const isWindows = () => {
+        return userAgent.includes('Windows');
+    };
+    const isMac = () => {
+        return userAgent.includes('Mac OS X');
+    };
+    const isLinux = () => {
+        return userAgent.includes('Linux');
+    };
+    const getDefaultShortcutForLetter = (event, letter) => {
```

Comment:
this name should not start with `get` because we don't get anything it check something so it should start with `is`